src/pages/academy/grading/Grading.tsx

```diff
@@ -165,7 +170,9 @@ const Grading: React.FC = () => {
                 <Button
                   minimal
                   icon={IconNames.EXPORT}
-                  onClick={() => exportGradingCSV(gradingOverviews.data)}
+                  onClick={() => {
+                    getAllGradingOverviews(tokens).then(resp => exportGradingCSV(resp?.data));
+                  }}
```

Comment:
Then, we use a non-null assertion instead (which should be safe since the page will auto-redirect if the token is not in the state)

```suggestion
                  onClick={() => {
                    const tokens = {
                      accessToken: accessToken!,
                      refreshToken: refreshToken!
                    };
                    getAllGradingOverviews(tokens).then(resp => exportGradingCSV(resp?.data));
                  }}
```